script/Ciel/Day 13/13_01_02_CIEL13_8B.txt

```diff
@@ -131,22 +157,23 @@ I hear the sound of clothes rustling.
 [sha:753fd2f41bf74308db5c3d32d22d9b9ebae0a3a0]{
 -- Page 516, Offset 36723.
 -- 外がまだ藍色のうちに先輩はベッドから出て、修道服姿に着替えていた。
-Senpai gets out of bed while the sky is still dyed in a light indigo, changing into her habit.
+The sky outside is still dyed a deep indigo,
+ but Senpai has already gotten up and is changing into her habit.
 }
 [sha:504df3a7a2e9c9069808781c97d1049f164762da]{
 -- Page 516, Offset 36724.
 -- その横顔を、ベッドに横になったまま見上げている。
-I look up at her profile while still lying on the bed.
+As I lie in bed, I gaze up at her silhouette.
 }
 [sha:6db8951a8fe03457aa5ba9b776dae81afe12683c]{
 -- Page 517, Offset 36725.
 -- 「……時間ですね。
-"...I guess it's time.
+"I guess it's time.
 }
 [sha:beb4cdee209a64c5870c3716ad524d7d7835102b]{
 -- Page 517, Offset 36726.
 -- 遠野くんをポケットにしまえたら、一緒にいられるのに」
-If only I could fit you inside my pocket, then we could've gone together."
+If you only fit in my pocket, I'd have been able to take you with me."
```

Comment:
I can't believe this line is actually MBTL foreshadowing.

With the way you changed the first clause, the second clause of the original TL makes more sense, but then the first clause of the original TL makes more sense with your second clause